publication-commons/src/main/java/no/unit/nva/publication/service/impl/UpdateResourceService.java

```diff
@@ -171,7 +170,7 @@ private TransactWriteItem updateResource(Resource resourceUpdate) {
             primaryKeyEqualityConditionAttributeValues(resourceDao);
 
         Put put = new Put()
-            .withItem(toDynamoFormat(resourceDao))
+                      .withItem(resourceDao.toDynamoFormat())
```

Comment:
Not sure why github shows that lines are not aligned here. They are in the code.